### ISSUE TITLE
Nodes crashing

### ISSUE THREAD

**Issue by Vibrio787** [seeleteam/go-seele#1055]
This is happening on multiple machines.  All run Ubuntu 18.04.  See attached terminal log.  

Please investigate.

[node crash.txt](https://github.com/seeleteam/go-seele/files/3867702/node.crash.txt)

**Comment by liangkenan** [comment]
原来遇到过这种问题，我的E5-2650 破烂cpu如果一台机器同时开4个节点，过几天就挂了，就是报这个错，后来我一台机器只开一个节点，这种情况会好很多，但是时间长了一些机器还是会出现这样的问题。

**Comment by Jimjimini** [comment]
Yes, the same with me, sometimes the nodes just stop, and I get multiple errors now, broken pipe etc, error writing TX, etc, all the errors listed here are now common place.

**Comment by ZenMaster1984** [comment]
Happened to my node 5 times in last 2 days
